### PR TITLE
ci: fix unittest autotune random fail

### PR DESCRIPTION
**Pull request by shjwudp**

### CODE REVIEW

**Comment by shjwudp** [comment]
The cause of the problem has been identified, and the small number of iterations leads to probabilistic failure to complete the entire autotune process.

File: tests/service/test_autotune_service.py [hunk 1]
```diff
@@ -51,7 +52,7 @@ def __init__(
         self.tensor_list = tensor_list
         self.client = AutotuneClient(service_addr, service_port)
 
-    def run(self, total_iters=5000):
+    def run(self, total_iters=500000):
```
Comment (NOBLES5E) [inline tests/service/test_autotune_service.py:55]: is the test deterministic now or you are trying to get a better result with more iterations?

**Review by NOBLES5E — CHANGES_REQUESTED** [review]
see comments

File: tests/service/test_autotune_service.py (hunk 1 above)
Comment (shjwudp) [reply tests/service/test_autotune_service.py:55]: test is deterministic, but need more iters to make sure autotune finishes.

**Pull request merged**